typescript/cli/tmp/anvil2/warp-route-deployment-anvil2.yaml

```diff
@@ -0,0 +1,3 @@
+anvil2:
```

Comment:
Did you mean to include these? If so, let's not put them in `cli/tmp`, the `tmp` makes it seem like they shouldn't be checked in

Reply:
sorry this was meant to be a draft to debug. Unfortunately i could not get this working with CI yet. The test hangs and then eventually fails (after 5 hours). 

I've concluded that the e2e tests do fails. My hunch is that it's anvil related, possibly conflicting with the foundry test script in the SDK. 

Will need more time to investigate the raw logs.

Reply:
Will remove